tests/test_main.py

```diff
@@ -1,6 +1,11 @@
+import mock
+import sys
+
 from validator.main import main
 
-def test_main():
+
+@mock.patch.object(sys, 'stderr')
```

Comment:
This isn't related to the PR, but this is the first time I noticed this problem, and I didn't want it in my test output.